cmd/controlplane/controller.go

```diff
@@ -142,6 +143,7 @@ func newControllerCommand() *cobra.Command {
 				argoClientForCreds = appMgr.GetClient()
 			}
 			credentialsDB := credentials.NewKubernetesDatabase(
+				strings.Split(os.GetEnv("KARGO_SHARED_CREDENTIALS_NAMESPACES", ""), ","),
 				os.GetEnv("ARGOCD_NAMESPACE", "argocd"),
```

Comment:
See [this comment](https://github.com/akuity/kargo/pull/1041/files#r1394755140)

This would become something like:

```golang
credentialsDB := credentials.NewKubernetesDatabase(
	credentials.KubernetesDatabaseConfigFromEnv(),
	kargoMgr.GetClient(),
	argoClientForCreds,
)
```